src/analysis/sum_stat.py

```diff
@@ -0,0 +1,205 @@
+import logging
+import os
+
+import pandas
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import subprocess
+import re
+import matplotlib.pyplot as plt
+
+logging.basicConfig(level=logging.INFO, format='%(levelname)s:%(asctime)s %(message)s')
+
+
+class SummaryStat(object):
+    '''
+    A class to easily aggregate multiple summary statistics files.
+    '''
+
+    def __search(self, regex: str, l: list, pvalue=False):
+        results_cords = []
+        results_names = []
+        for i, name in enumerate(l):
+            if re.search(regex, name):
+                if pvalue:
+                    results_cords.append(i + 1)
+                    results_names.append(name)
+                else:
+                    return i + 1, name
+        if pvalue:
+            return results_cords, results_names
+        return False, False
+
+    def __get_ss_info(self, header: list, tag: str):
```

Comment:
please add docs

Reply:
I don't think providing docs for private methods is necessary, because they won't be directly used and i made them only to increase main function (`prepare_data`) readability.